airflow/providers/common/sql/CHANGELOG.rst

```diff
@@ -32,7 +32,9 @@ This release fixes a few errors that were introduced in common.sql operator whil
 * ``_process_output`` method in ``SQLExecuteQueryOperator`` has now consistent semantics and typing, it
   can also modify the returned (and stored in XCom) values in the operators that derive from the
   ``SQLExecuteQueryOperator``).
-* last description of the cursor whether to return scalar values are now stored in DBApiHook
+* descriptions of all returned results are stored as descriptions property in the DBApiHook
```

Comment:
(could be done later) maybe it would be a bit easier to read if `descriptions` and `last_description` are formatted as code?

Reply:
Yep. We can update it later :)